app/controllers/admin/power_users_controller.rb

```diff
@@ -1,7 +1,5 @@
 module Admin
   class PowerUsersController < BaseController
-    before_action :set_user, only: %i[resend_confirmation destroy]
-
     def index
       @users = policy_scope([:admin, :power_users, User.all])
```

Comment:
Il y a un problème de performance (requête N+1) dans la vue correspondante.
Cette suggestion permet de la résoudre en préchargeant les roles pour tous les users en une seule requête SQL 

```suggestion
      @users = policy_scope([:admin, :power_users, User.includes(:roles)])
```

Reply:
le code a bougé depuis, je suis curieux d'avoir ton avis

Reply:
@carsso en l'état la requête n+1 a encore lieu, après nouvelle revue.
Ici : https://github.com/hostolab/covidliste/pull/457/files#diff-8b210d967bbf202a145822d4db96f2a04f35b9396a2aa60db436898b49a3ab8fR42
Et là : https://github.com/hostolab/covidliste/pull/457/files#diff-8b210d967bbf202a145822d4db96f2a04f35b9396a2aa60db436898b49a3ab8fR59

Elle est mitigée par `pluck`, qui dans notre cas sélectionne uniquement les ids, mais du coup il est mieux de précharger les `roles` une bonne fois pour les avoir en mémoire.